sphinx_needs/need_constraints.py

```diff
@@ -45,6 +46,11 @@ def process_constraints(app: Sphinx, need: NeedsInfoType) -> None:
                 need["constraints_results"].setdefault(constraint, {})[name] = False
                 need["constraints_passed"] = False
 
+                if "error_message" in executable_constraints:
+                    need["constraints_error"] = jinja2.Template(str(executable_constraints["error_message"])).render(
```

Comment:
Initiating a jinja template takes some time, can we do it only once and execute only the rendering per need?